### DISCUSSION TITLE
Ideas for saving resources when not needed

### DISCUSSION THREAD

**Discussion by GGamer6458** [ppy/osu#23273] [Ideas] [answered]
I've noticed certain scenarios within the game where it would be handy to limit the framerate so it doesn't waste resources. Basically just any point where you aren't actively playing the game or where low-latency is not needed.

Some scenarios where I would like to see the framerate be limited in are:

- During breaks
- [In the menu](https://github.com/ppy/osu/discussions/22364#discussioncomment-4762876)
- In the editor 
- While the game is paused
- During inactivity
- Lowering the priority of the game when in the background

I would love to see power saving features like this built into the game, especially when you play on laptops and mobile devices where power saving and not wasting resources is very important.

**Comment by Theighlin** [comment]
The game already locks every thread to 60fps when in the background, you can see the difference in resources used in task manager

**Reply by GGamer6458** [reply]
I am aware of that, but locking the thread is different from lowering the process priority

**Comment by peppy** [comment, marked as answer]
Windows already drops process priority when not in focus. This is called "priority boosting".

If you're using "basically unlimited", the solution is to not use it. We'll probably eventually limit menus etc. similar to stable though, even for unlimited.
